.github/workflows/e2e_tests.yaml

```diff
@@ -7,11 +7,11 @@ name: CI
 on:
   push:
     branches:
-      - dev-next
+      - dev-next-fornax
       - 'CentaurusInfra/mizar'
   pull_request:
     branches:
-      - dev-next
+      - dev-next-fornax
```

Comment:
I'm concerned that we might forget to update this when we merge to dev-next.
Can we track this as an issue to fix? Or preferably create a new file e2e_tests_fornax (if that works) and track to remove it before merge?

Reply:
If we create a new file e2e_tests_fornax, we will have to synchronize all the changes in the e2e_tests in future. An easy way to is add dev-next-fornax to the trigger branches. The 'CentaurusInfra/mizar' is not a valid branch. I replaced it with dev-next-forna.